plugins/obs-webrtc/webrtc/src/output.rs

```diff
@@ -67,12 +74,86 @@ pub struct OutputStream {
     peer_connection: Arc<RTCPeerConnection>,
     stats: Arc<RwLock<OutputStreamStats>>,
     worker_tx: UnboundedSender<Message>,
-    worker_future: Arc<Mutex<Option<JoinHandle<()>>>>,
+    worker_handle: Arc<Mutex<Option<JoinHandle<()>>>>,
     whip_resource: Arc<Mutex<Option<Url>>>,
     error_callback: Arc<Mutex<Option<ErrorCallback>>>,
 }
 
 impl OutputStream {
+    fn start_worker(&mut self, mut worker_rx: UnboundedReceiver<Message>) {
+        let worker_handle = std::thread::spawn({
+            let audio_track = self.audio_track.clone();
+            let video_track = self.video_track.clone();
+            let peer_connection = self.peer_connection.clone();
+            let stats = self.stats.clone();
+            let error_callback = self.error_callback.clone();
+            move || {
+                let runtime = tokio::runtime::Builder::new_multi_thread()
+                    .enable_all()
+                    .thread_name("webrtc_worker")
+                    .worker_threads(4)
+                    .build()
+                    .unwrap();
+                let runtime = runtime;
+                let result = runtime.block_on({
```

Comment:
the extra `{` (+ corresponding `}`) here is unnecessary/superfluous now.

Reply:
Fixed in e5854aa721a1235c8ca72bec4cdf5293ab0f781b